config.toml

```diff
@@ -1,42 +1,16 @@
-iso = 'Windows7Ultimate.iso'
-network = 'Pool-wide network associated with eth0'
-pool = 'lab3'
-pvVm = 'xo-test-pv'
-vmToMigrate = 'souad'
-xoServerUrl = 'localhost:9000'
-vmIdXoTest = '0342407f-9727-4f12-83c2-9d0c458916c2'
-xoTestTemplateId = '1a33d04d-1a9c-e2af-ae42-010554eed513'
-
-# powerOnMode has to be not empty
-host1 = 'lab1'
-
-[adminCredentials]
+[xoConnection]
+  url = 'localhost:9000'
   email = 'admin@admin.net'
   password = 'admin'
 
+[vms]
+  default = '0342407f-9727-4f12-83c2-9d0c458916c2'
```

Comment:
I think we should change the `vmId` because it no longer exists and that can be a problem for some tests.

Reply:
This PR just structure the `config.toml` but i can add the new VM id if you want. 
We should remove entries values when integrating `xo-server-test` to `xen-orchestra` because it's proper to the user.